app/src/main/java/nl/rijksoverheid/en/settings/ExposureNotificationsPausedReminderReceiver.kt

```diff
@@ -22,7 +23,7 @@ import java.time.ZoneId
 class ExposureNotificationsPausedReminderReceiver : BroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
         Timber.d("onReceive")
-        NotificationsRepository(context).showAppPausedReminder()
```

Comment:
In this case the repository is created only on this line, so I don't think it's needed to extract a function for it. In `EnWorkerFactory` I did do that because we have multiple places where that repository is being passed in and it seemed easier and safer to have a separate function to create it in that case, should we need to update the creating of that thing in the future for some reason.